src/TcUnit.TestAdapter/Models/TmcModule.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Xml.Linq;
+
+namespace TcUnit.TestAdapter.Models
+{
+    public class TmcModule : TmcItem
+    {
+        public List<TmcDataArea> DataAreas { get; set; } = new List<TmcDataArea>();
+
+        public string TcSmClass { get; set; }
+
+        internal TmcModule(string name, string tcSmClass)
+        {
+            Name = name;
+            TcSmClass = tcSmClass;
+        }
+
+        public static TmcModule Parse(XElement element)
+        {
+            string name = element.Element("Name").Value;
+            string tcSmClass = element.Attribute("TcSmClass").Value;
+
+            var module = new TmcModule(name, tcSmClass);
+
+            var dataAreas = element.Elements("DataAreas")
+                                    .Elements("DataArea");
+            foreach (var dataArea in dataAreas)
+            {
+                module.DataAreas.Add(TmcDataArea.Parse(dataArea));
```

Comment:
More testing should be done on projects that contain multiple PLCs or other types of TwinCAT project like C++ and Vision, as I suspect these create other modules in the TMC file.